pkg/services/serviceaccounts/database/database.go

```diff
@@ -132,33 +139,48 @@ func (s *ServiceAccountsStoreImpl) ListTokens(ctx context.Context, orgID int64,
 	return result, err
 }
 
-func (s *ServiceAccountsStoreImpl) ListServiceAccounts(ctx context.Context, orgID, serviceAccountID int64) ([]*models.OrgUserDTO, error) {
+func (s *ServiceAccountsStoreImpl) ListServiceAccounts(ctx context.Context, orgID, serviceAccountID int64) ([]*serviceaccounts.ServiceAccountDTO, error) {
 	query := models.GetOrgUsersQuery{OrgId: orgID, IsServiceAccount: true}
 	if serviceAccountID > 0 {
 		query.UserID = serviceAccountID
 	}
-
 	err := s.sqlStore.GetOrgUsers(ctx, &query)
 	if err != nil {
 		return nil, err
 	}
-
-	return query.Result, err
+	saDTOs := make([]*serviceaccounts.ServiceAccountDTO, len(query.Result))
+	for i, user := range query.Result {
+		saDTOs[i] = &serviceaccounts.ServiceAccountDTO{
+			Id:    user.UserId,
+			OrgId: user.OrgId,
+			Name:  user.Name,
+			Login: user.Login,
+		}
+		tokens, err := s.ListTokens(ctx, user.OrgId, user.UserId)
```

Comment:
one more call for each service account to the db, which is not optimal.

however it is at most 20 service account per customer